api/src/main/resources/client_api.yaml

```diff
@@ -74,6 +74,70 @@ paths:
           schema:
             $ref: "#/definitions/MaterializeCohortResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/concepts:
+      post:
```

Comment:
Wasn't this a GET previously?

If you're going to make it a POST, then you should use a custom verb at the end of the URL, e.g. `.../{workspaceId}/searchConcepts` or `.../{workspaceId}/concepts:search`. POST to a noun (what you have currently) generally means "create that noun"

Reply:
Done.